src/shell.js

```diff
@@ -371,8 +371,8 @@ if (ENVIRONMENT_IS_WEB || ENVIRONMENT_IS_WORKER) {
 #if MODULARIZE
   // When MODULARIZE, this JS may be executed later, after document.currentScript
   // is gone, so we saved it, and we use it here instead of any other info.
-  if (_scriptDir) {
-    scriptDirectory = _scriptDir;
+  if (_scriptName) {
+    scriptDirectory = _scriptName;
```

Comment:
Perhaps this should also be renamed? Right now this line reads as "dir = name"

Reply:
In this case it actually makes sense to do that because the next line down effectively does `os.path.dirname` on it.  So this is just an intermediate value for `scriptDirectory` that is about to be changed.

Reply:
I could create a new intermediate variable here that as a better name, but that kind of goes against of style of minimizing code size.

Reply:
I see, makes sense.